test/junit/scala/concurrent/duration/DurationTest.scala

```diff
@@ -0,0 +1,27 @@
+package scala.concurrent.duration
```

Comment:
Yep. Go with Viktor's suggestion of putting them in duration-tck.scala. Sorry for misguiding you. And thank you for the contribution.